server_src/cache/printer.cache.js

```diff
@@ -0,0 +1,36 @@
+const { PrinterClean } = require("../lib/dataFunctions/printerClean");
+
+let printerCleanState = new PrinterClean();
+
+function getPrinterCache() {
+  return printerCleanState;
+}
+
+function returnPrintersInformation() {
+  return printerCleanState.printersInformation;
+}
+
+function returnPrinterControlList() {
+  return printerCleanState.printerControlList;
+}
+
+function returnFilamentList() {
+  return printerCleanState.printerFilamentList;
+}
+
+function returnCurrentOperations() {
+  return printerCleanState.currentOperations;
+}
+
+function returnDashboardStatistics() {
+  return printerCleanState.dashboardStatistics;
+}
+
+module.exports = {
```

Comment:
Is there a reason for grouping all the exports here? 

I've opted for exporting at the function level. If it's a preference rather than anything functional then I'm happy to switch mine out and do as an export at the bottom. Just want to keep these consistent.

Reply:
I thought about it a bit, when we switch to Typescript all these functions will go into a class and that class will be prepended with `export`. So having to delete one block at the bottom with module.exports is easiest.
Also, I find the explicit block at the bottom much cleaner.

Reply:
Sound I'll switch mine out to a single export at the bottom.